model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -128,7 +128,13 @@ class LicenseInfoResolver(
         val unmatchedCopyrights = mutableMapOf<Provenance, MutableSet<CopyrightFinding>>()
         val resolvedLocations = resolveLocations(filteredDetectedLicenseInfo, unmatchedCopyrights)
         val detectedLicenses = licenseInfo.detectedLicenseInfo.findings.flatMapTo(mutableSetOf()) { findings ->
-            findings.licenses.map { it.license }
+            val curatedFindings = FindingCurationMatcher().applyAll(
+                findings.licenses,
+                findings.licenseFindingCurations,
+                findings.relativeFindingsPath
+            ).mapNotNull { it.curatedFinding }
+
+            curatedFindings.map { it.license }
```

Comment:
Could be merged into the above expression for less iterations: `mapNotNull { it.curatedFinding?.license }`

Reply:
IMO keeping the variable helps with readability. Do you agree?

Reply:
Yes, but I'd still like to avoid the duplicate iteration. So how about moving the `mapNotNull` down to `curatedFindings.map { it.curatedFinding?.license }` (and maybe renaming `curatedFindings` to `curationResults` then)?

Reply:
I ended up going for your original suggestion.